src/Components/Backoffice/Home/Edit.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { useFormik } from "formik";
+import Button from "@material-ui/core/Button/Button";
+import EditItem from "./EditItem";
+
+const Edit = (props) => {
```

Comment:
The name is not descriptive enought, what are you editing? what type of component is this? I think that the name should be maybe EditHomeForm

Reply:
About adding "Home", I thought that having the Edit component inside the home folder would make it redundant to name it EditHome. I'll change it anyways to EditHomeForm.